### PR TITLE
Feature: Allow Admin/SuperAdmin Sign In

### PR DESCRIPTION
**Pull request by Saint-lawal**
### What does this PR do?
- Allow admin/superamin sign in.
- Also verify superadmin's phone number and email by default.

### CODE REVIEW

File: controllers/Auth.js [hunk 1]
```diff
@@ -136,8 +136,7 @@ class Auth {
         // check for an existing user
         user = await db.User.findOne({
           email,
-          isDeleted: false,
-          role: 'user'
+          isDeleted: false
```
Comment (adetola-ralph) [inline controllers/Auth.js:139]: `role: 'user'` is necessary for this controller method. @bolorundurowb added another route and controller for admins to login. Check [here](https://github.com/Our-Leaders/our-leaders-api/blob/develop/controllers/Auth.js#L206) for the admin login controller handler

Comment (Saint-lawal) [reply controllers/Auth.js:139]: This is to allow admins and superadmins login as users on the client side. @adetola-ralph

Comment (adetola-ralph) [reply controllers/Auth.js:139]: Is there any reason why would we want that?

Comment (Saint-lawal) [reply controllers/Auth.js:139]: He mentioned it here https://trello.com/c/CYdYeRXU and somewhere else that I can't find at the moment

Comment (bolorundurowb) [reply controllers/Auth.js:139]: @Saint-lawal we addressed Nosas requirements by allowing for the same email address to be used for an admin and for a user. An admin can signup as a user with the same email address. Like @adetola-ralph mentioned I added an endpoint to log admins in

Comment (Saint-lawal) [reply controllers/Auth.js:139]: 👍

**Pull request closed without merging**